### ISSUE TITLE
Influxdb-admin unreachable at 8083

### ISSUE THREAD

**Issue by anarwal27** [influxdata/influxdb#8541]
I used to run influxdb-admin by providing property: `ADMIN_ENABLED: 'true'` and it used to work fine, however now I am unable to access admin on 8083. has there been any updates for admin accessibility?

**Comment by e-dard** [comment]
@anarwal27 the Admin UI has been deprecated and removed [as of 1.3](https://github.com/influxdata/influxdb/blob/1.3/CHANGELOG.md#removals).
